.gitignore

```
@@ -233,3 +233,4 @@ $RECYCLE.BIN/
 # Windows shortcuts
 *.lnk
 *.pp
+/nugets/**
```

Change to `dist` (see comment later as to why):

```suggestion
/dist/**
```